source/common/upstream/BUILD

```diff
@@ -426,6 +428,7 @@ envoy_cc_library(
         "//source/common/protobuf:utility_lib",
         "//source/common/runtime:runtime_lib",
         "//source/server:transport_socket_config_lib",
+        "//source/extensions/upstream/local_address_selector:config",
```

Comment:
hmmm... depending on non-core extension should be discouraged. Is this really needed? Is there a way to work with interfaces here?